### ISSUE TITLE
Can you make separate folders in the mods folder for the different updates so its easy to differentiate?

### ISSUE THREAD

**Issue by Pandorify** [HolographicWings/TOTK-Mods-collection#34]
Whenever the 1.1.1 updated mods start being here could they be moved to a separate 1.1.1 folder, and a folder for 1.1.0 and maybe 1.0.0? I think it would be nice to be able to just see the mods for one update. It would also be nice if the categories like fps, visual, and cheats were in different folders in their respective update folders inside the zip. Sorry if this isn't the right place to ask this, I don't use GitHub much so I don't know how this usually works... Thanks!

**Comment by HolographicWings** [comment]
We don't want to made several root folders to don't confuse people, it's why i added a prefix to every mod with the version number.

**Comment by Pandorify** [comment]
For me so many files in one folder confuses me a bit since there are just
so many, though I see why other people might find them all being in one
place helpful. Would it be possible to have two releases that have them
organized in different ways?

**Comment by Krutonium** [comment]
Personally I agree - Having a folder structure that contains mods sorted by game version would certainly not be a bad thing imo.

**Comment by Tokinsom** [comment]
Yeah starting the names with the game version(s) is not a good idea. The groups are all scattered now. Would be best to have different folders for each game version.

**Comment by HolographicWings** [comment]
We will think about, the risk is to get issue reports because someone dropped the 1.1.1 folder in his mod folder by exemple.
Yuzu would need a PR to read subfolders tho.

**Comment by Krutonium** [comment]
> We will think about, the risk is to get issue reports because someone dropped the 1.1.1 folder in his mod folder by exemple. Yuzu would need a PR to read subfolders tho.

No it wouldn't?

The structure would be

```
/TOTK-Mods-collection/1.1.1/<mod folders here>
/TOTK-Mods-collection/1.1.0/<mod folders here>
```

**Comment by ctparton** [comment]
I agree with @Krutonium suggestion, this structure would make far more sense as the root folder is quite hard to parse right now

**Comment by HolographicWings** [comment]
The issue is that will obligate us to duplicate some mods
Some mods are made for every versions, we will be obligated to drop them in the three folders then, and will be boring to update them.

**Comment by Pandorify** [comment]
Would putting cheats, mods, and the combo packs into different root folders
be any easier? I think that would improve it a lot even if they weren't
separated by version

**Comment by Pandorify** [comment]
Oh, I didn't see that it was already changed to that lol, it's a lot better now. Thanks!
